UnitedIncome/components
# Color Story

### Description
> Please provide a brief description of your changes below.

I had a few minutes at the end of my day so I added a `colors` story that gets imported from the `colors.js` file. This allows us to track the colors using Percy which is ideal for rebranding.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Go to utilities > styles > colors.

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

<img width="2672" alt="Screen Shot 2020-01-10 at 4 22 29 PM" src="https://user-images.githubusercontent.com/10888441/72187402-8ae41a00-33c5-11ea-8256-563e4b44102a.png">

### Comments
> Any additional comments that will aid with the review process.

I'd like to extend this pattern to all of our style stories. I'd also like to eventually make the JS file the one source of truth in this library, but that can be a change for another day.
